website/cue/reference/components/sources/host_metrics.cue

```diff
@@ -116,6 +117,72 @@ components: sources: host_metrics: {
 				unit:    "seconds"
 			}
 		}
+		cgroups: {
+			common:      false
+			description: #"Options for the "cgroups" metrics collector. This collector is only available on Linux systems, as the cgroups (control groups) system is only present on Linux."#
```

Comment:
I think we should make it clear that we only support cgroups v2 / unified (at least for now).

Reply:
Agreed. Addressed in 2e4c2a271